src/i18n/en-US/governanceReviewTeam.ts

```diff
@@ -93,6 +93,12 @@ const governanceReviewTeam = {
       requestType: 'Type of request',
       submissionDate: {
         null: 'Not yet submitted'
+      },
+      grtDate: {
+        null: 'Add date'
```

Comment:
I thought about making these the same but wanted to leave them separate in case we wanted to change it to "Add GRB Date" or something similar instead. Im not opposed to combining them.

Reply:
Since they're currently the same text, it's better to keep one copy of it. Your forward thought is possible, but it's not an issue and it's fairly easy to revert.

The other thing is that we're getting into a pattern of mapping our i18n keys to the database values. This can get messy quickly since our text is entangled with the enums we store rather than having a separation of concerns.